web/src/personal_menu_popover.js

```diff
@@ -58,9 +58,7 @@ export function initialize() {
 
             $popper.one("click", ".personal-menu-clear-status", (e) => {
                 e.preventDefault();
-                const me = current_user.user_id;
                 user_status.server_update_status({
-                    user_id: me,
```

Comment:
https://zulip.com/api/update-status

Reply:
That makes sense, you can't edit them for other users.